exercises/practice/paasio/src/test/java/PaasioTest.java

```diff
@@ -0,0 +1,337 @@
+import org.junit.jupiter.api.*;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class PaasioTest {
+
+    private ByteArrayInputStream dataInputStream;
+    private ByteArrayOutputStream dataOutputStream;
```

Comment:
Since every test creates and uses a Paasio object, we could also use a Passio field and set it in `setUpTest` and then close it after every test in a `@AfterEach` method.